modules/understanding-the-administration-event-details.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// *operating/using-the-administration-events-dashboard.adoc
+
+:_content-type: CONCEPT
+[id="understanding-the-administration-event-details_{context}"]
+= Understanding the administration event details
+
+The administration event details page organizes information in the following groups:
+
+* *Error*: Provides log information that explains the nature of the error or event. It provides important information for understanding the specific issue or incident. This field contains information about the context of the event and the steps you need to take to fix the error.
+* *Resource type*: Classifies events based on the type of resource or component involved. Resource types include `API Token`, `Cluster`, `Image`, `Node`, and `Notifier`.
+* *Resource name*: Specifies the name of the resource or component to which the event refers. It identifies the specific instance or entity within the system where the event occurred.
+* *Event type*: Categorizes the nature of the event. It describes the type of action or incident that took place.
```

Comment:
Event type currently only specifies _where_ the event is coming from, and today only "Log" events are being created by Central, meaning administration events created from log statements that have ocurred.